src/ImageRunModal.jsx

```diff
@@ -652,7 +652,7 @@ export class ImageRunModal extends React.Component {
         // Add the search component
         const footer = (
             <ToggleGroup className='image-search-footer' aria-label={_("Search by registry")}>
-                <ToggleGroupItem text={_("All")} key='all' isSelected={this.state.searchByRegistry == 'all'} onChange={(_, ev) => {
+                <ToggleGroupItem text={_("All")} key='all' isSelected={this.state.searchByRegistry == 'all'} onChange={(ev, _) => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1322-20230703-053724-7cc9e4bf-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)